tests/pytest/README.rst

```diff
@@ -0,0 +1,38 @@
+Hypothesis has excellent integration with ``py.test``, though obviously
```

Comment:
nit: not a fan of "obviously" in technical documentation.

If it was really obvious, we wouldn't say it. If it's not obvious, we're just making somebody who didn't realise that feel bad.